test/LocationTest.cpp

```diff
@@ -0,0 +1,17 @@
+//Spatial/Location Test
+
+#include "Spatial/Location.h"
+#include "gtest/gtest.h"
+
+using namespace Spatial;
+
+TEST(LocationTest, LocTest) {
+Location loc(1, 0.123, 3.456, 10000);
+
+ASSERT_EQ(1, loc.id);
+ASSERT_EQ(0,loc.beta);
+ASSERT_EQ(10000,loc.population_size);
+//ASSERT_EQ(Approx(0.123),loc.coordinate->latitude);
```

Comment:
Try to test if the lat and longitute was inintialized with the right value

Reply:
longitude and latitude is not working. Approx() is not recognized.

Reply:
The point is how to compare floating point number in C++. 
You can refer to this: 
https://testing.googleblog.com/2008/10/tott-floating-point-comparison.html

Reply:
Thanks.